### ISSUE TITLE
Target output does not respect base

### ISSUE THREAD

**Issue by joshuaavalon** [addyosmani/critical#356]
Version: v2.0.0-12

```
project/
└─── dist/
```

```js
critical.generate({
  base: "dist",
  src: "index.html",
  target: "index.html",
  inline: true,
  minify: true,
  extract: true,
  penthouse: {
    blockJSRequests: false
  }
});
```

The current directory is `project`.

**Expected Behavior**
`index.html` is replaced.

**Current Behavior**
New `index.html` is located at `project` instead of `project/dist`.

**Comment by bezoerb** [comment]
@joshuaavalon: Should be fixed with `v2.0.0-13`

**Comment by joshuaavalon** [comment]
Confirmed it works in `v2.0.0-13`.
